src/landscape/components/LandscapeList.js

```diff
@@ -44,21 +49,44 @@ const MemberLeaveButton = withProps(LandscapeMemberLeave, {
   label: 'landscape.list_leave_button',
 });
 
-const MemberJoinButton = withProps(GroupMemberJoin, {
+const MemberJoinButton = withProps(MemberJoin, {
   label: 'landscape.list_join_button',
   ariaLabel: 'landscape.list_join_label',
 });
 
-const MembershipButton = ({ landscape, tabIndex }) => (
-  <GroupContextProvider
-    owner={landscape}
-    groupSlug={_.get('defaultGroup.slug', landscape)}
-    MemberJoinButton={MemberJoinButton}
-    MemberLeaveButton={MemberLeaveButton}
-  >
-    <GroupMembershipJoinLeaveButton tabIndex={tabIndex} />
-  </GroupContextProvider>
-);
+const MembershipButton = ({ landscape, tabIndex }) => {
```

Comment:
question: can we refactor/reuse what's in GroupList?

Reply:
@paulschreiber Do you have a suggestion of what can be reused between the 2?